### ISSUE TITLE
508-defect-2 [KEYBOARD]: Focus should be placed on the `h3` when navigating backwards to the first slide in the VR&E Orientation slideshow

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22663]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement
As a keyboard user, I expect focus to be consistently managed within the VR&E Orientation slideshow

## Details

When a keyboard user returns back to the first VR&E Orientation slide using the "Previous Slide" button, focus is placed somewhere within the Orientation group but not on the `h3` "Veteran Readiness and Employment orientation." Placing it directly on the `h3` creates consistency within the slideshow experience.

## Acceptance Criteria

- [x] Upon pressing the "Previous slide" button on slide 2, the user is returned to slide 1 with focus on the `h3` "Veteran Readiness and Employment orientation"

## Screenshots or Trace Logs

Currently, focus is lost within the group

https://user-images.githubusercontent.com/14154792/113624534-c6ec4600-962d-11eb-81af-5d382c5ce8cb.mov

**Comment by jason-gcio** [comment]
Might only be slide 2?

**Comment by joshkimux** [comment]
@jason-gcio , correct, I noted in the title it occurs when navigating backwards to the first slide 😄

**Comment by jason-gcio** [comment]
Thanks @joshkimux - I was leaving ourselves a note haha we are working on it

**Comment by jason-gcio** [comment]
@joshkimux - would you like to take a look at this before closing?

**Comment by joshkimux** [comment]
<img width="792" alt="Screen Shot 2021-05-03 at 12 19 41 PM" src="https://user-images.githubusercontent.com/14154792/116902996-d8634680-ac09-11eb-971d-69ee93a80973.png">

It's been fixed! Great job team 🥳
